src/utils/database-common/src/db_provider.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug)]
+pub enum DatabaseProvider {
+    Postgres,
+    MySql,
+    MariaDB,
+}
```

Comment:
Why `SqLite` is not on this list?

Reply:
I've skipped binding to CLI experiment as boring.
Okay, added for consistency.